prisma/schema.prisma

```diff
@@ -0,0 +1,36 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Keyword {
+  id                   Int           @id @default(autoincrement())
+  keyword_name         String        @unique
```

Comment:
To follow JS camelcase naming, the following can be done with Prisma:

```
keywordName  String?  @map("keyword_name")
```

Thus, the attribute name follows DB conventions, while the model attribute follows JS conventions.